src/components/Badge/BadgeReport.jsx

```diff
@@ -328,6 +331,19 @@ const BadgeReport = props => {
                 ) : (
                   []
                 )}
+                <td>
+                  <Dropdown>
+                    <Dropdown.Toggle variant="secondary" id="dropdown-basic">
+                      Earned Dates
+                    </Dropdown.Toggle>
+
+                    <Dropdown.Menu>
+                      {value.earnedDate.map((item, i) => (
```

Comment:
Hey, @johny4437 while looking trying to review this the badge report modal window would crash the app. I think I tracked it down to this line, value does not have an earnedDate property. I removed "value." from the line and was able to see the modal window, however interacting with this dropdown menu would also crash the app.

Reply:
You need to test this branch with the backend branch too, if youy don't,  won't work.